samples/python/benchmark/throughput_benchmark/README.md

```diff
@@ -21,7 +21,7 @@ The following Python API is used in the application:
 
 | Feature                   | API                                             | Description                                  |
 | --------------------------| ------------------------------------------------|----------------------------------------------|
-| OpenVINO Runtime Version  | [openvino.runtime.get_version]                  | Get Openvino API version.                    |
+| OpenVINO API Version      | [openvino.__version__]                          | Get Openvino API version.                    |
```

Comment:
```suggestion
| OpenVINO API Version  | [openvino.__version__]  | Get Openvino API version.                    |
```

Reply:
Could you please remove unnecessary spaces

Reply:
I think we also need to avoid `runtime` namespace below

Reply:
@rkazants generally yes, but there is a task for doc team to update all doc files :)